templates/ruby/lib/http_client/error_handler.rb

```diff
@@ -27,7 +27,7 @@ def call(env)
             end
 {{if .Api.Response.Formats.Json}}
             # If JSON, a particular field is taken and used
-            if type.include?("json") and body.is_a?(Hash)
+            if type && type.include?("json") && body.is_a?(Hash)
```

Comment:
FYI `&&` and `and` does same thing but different precedence. It is not a problem here but can be confusing so [rubocop banned `and`](https://github.com/bbatsov/ruby-style-guide#no-and-or-or) for instance.